utils.py

```diff
@@ -108,6 +111,76 @@ def plot_stay_probability(data):
     # Show the plot
     plt.show()
 
+def preprocess_human_data(data_df: pd.DataFrame) -> pd.DataFrame:
+    data = data_df.copy()
+    # infer common transition from the action taken in stage 1 and isHighProbOne/Two
+    data['common_transition'] = np.where(data['stepOneChoice'] == 0,
+                                            data['isHighProbOne'],
+                                            data['isHighProbTwo'])
+
+    # infer the transition to stage 2 state from the action taken in stage 1 and isHighProbOne/Two
+    # update the state_transition_to value only if condition is met, else keep it as it is
+    data['state_transition_to'] = np.where((data['stepOneChoice'] == 0) & (data['isHighProbOne'] == True),
+                                        1,
```

Comment:
why do we use 1 and 2 instead of 0 and 1?

Reply:
the environment is modeled as 3 states ( 0: first stage , 1: second stage left, 2: second stage right ),
in each stage there's 2 action ( 0: left, 1: right ),
so in this case 1 and 2 refer to the second stage states